app/helpers/application_helper.rb

```diff
@@ -10,4 +10,8 @@ def resource
   def devise_mapping
     @devise_mapping ||= Devise.mappings[:user]
   end
+
+  def user_admin?
+    current_user.role.eql? "admin"
```

Comment:
What happens in case when current_user is nil ?

Reply:
also move this logic to applicaton_controller and user method "helper_method" to make it like helper https://apidock.com/rails/ActionController/Helpers/ClassMethods/helper_method

Reply:
Why should we carry about current_user == nil when we have before_action :authenticate_user! in controller? In what cases is it possible? And does "rework request' label means to cancel current reques and open a new one?

Reply:
As for check for nil. I'm OK to skip it.
But creating helper method is required

Reply:
Thanks, transformed "user_admin" method to applicaton_controller helper_method.